values/istio-base/istio-base.gotmpl

```diff
@@ -0,0 +1,10 @@
+{{- $v := .Values }}
+{{- $i := $v.apps.istio }}
+
+{{- with .Values.otomi | get "globalPullSecret" nil }}
+global:
+  imagePullSecrets:
+    - otomi-pullsecret-global
+{{- end }}
+
+defaultRevision: {{ $i.defaultRevision | default "default" | quote }}
```

Comment:
why having fallback value if you already defined it in the defaults.yaml?

Reply:
It is not a fallback, but a replacement for a blank string. Throughout the Istio templates, it varies what is needed. For example, `revision` should be set to a blank string if not used, but sometimes `default` needs to be set explicitly.